cmd/docker/docker.go

```diff
@@ -432,3 +435,14 @@ func hasTags(cmd *cobra.Command) bool {
 
 	return false
 }
+
+func isNoRemote(cmd *cobra.Command) bool {
+	for cmd != nil {
+		if _, ok := cmd.Annotations["no-remote"]; ok {
```

Comment:
nit, but I don't like negations in names, because you can end up with a double negations, like `if !isNoRemote(...)` ... and double negations are generally brain fucked 😅 
That said, I don't have better name to propose... I thought about "local" but I think it is too vague.
@thaJeztah any idea? 😅

Reply:
`IsLocal` ?

Reply:
IsDaemonLess ?

Reply:
Can you rename the annotation too? `no-remote` -> `local-only` ?